certified-connectors/Witivio/apiDefinition.swagger.json

```diff
@@ -3,7 +3,12 @@
   "info": {
     "title": "Witivio",
     "description": "Witivio 365 and Microsoft Flow.",
```

Comment:
@tdechanterac As per the submission you made in the ISV Studio, it should be - `"description": "Build advanced virtual agent with the power of Witivio 365"`

Reply:
Thank you. I've made the proper change.